### PR TITLE
Tools code coverage

### PR DESCRIPTION
**Pull request by peio42**

### CODE REVIEW

**Comment by axel3rd** [comment]
@peio42 : 0% of code coverage on modified code 😢 .... Did you want a validation of review ?

File: .travis.yml [hunk 1]
```diff
@@ -13,4 +13,3 @@ install:
 script:
   - npm test
   - sh sonar-project-update-props.sh && sonar-scanner
-
```
Comment (axel3rd) [inline .travis.yml]: A YAML file should have an empty line at end of file.

**Comment by peio42** [comment]
Sorry for the noise, I got good result from Travis and didn't wait for sonarcloud.
I put back the empty line but Sonarcloud is still failing...

**Comment by peio42** [comment]
Tools scripts are loaded by Javascript function "vm.runInContext" for the tests to hook inside function.
This way of loading isn't catched by "nyc", unless specific parameter is given, but it then breakes other test scripts.
Work in progress...

**Review by axel3rd — APPROVED** [review]

**Pull request merged**